### PR TITLE
[MINIAPPS: pt.2] feat: DOMA-4639 Added labels and displayPriority

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: apps/condo/domains/miniapp/schema/AllMiniAppsService.js [hunk 1]
```diff
@@ -2,12 +2,30 @@
...
  */
 
+const dayjs = require('dayjs')
 const { GQLCustomSchema } = require('@open-condo/keystone/schema')
 const access = require('@condo/domains/miniapp/access/AllMiniAppsService')
 const { ACQUIRING_APP_TYPE, BILLING_APP_TYPE, APP_TYPES, B2B_APP_TYPE } = require('@condo/domains/miniapp/constants')
 const { find } = require('@open-condo/keystone/schema')
 const { APPS_FILE_ADAPTER } = require('@condo/domains/miniapp/schema/fields/integration')
 
+const priorityCompare = (lhs, rhs) => {
```
Comment (sitozzz) [inline apps/condo/domains/miniapp/schema/AllMiniAppsService.js:12]: `lhs` and `rhs` is a little unclear name

File: apps/condo/migrations/20221202114207-0195_acquiringintegration_displaypriority_and_more.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
...ICAgIG5hbWU9J2Rpc3BsYXlQcmlvcml0eScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5JbnRlZ2VyRmllbGQoZGVmYXVsdD0xKSwKICAgICAgICAgICAgcHJlc2VydmVfZGVmYXVsdD1GYWxzZSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2JpbGxpbmdpbnRlZ3JhdGlvbicsCiAgICAgICAgICAgIG5hbWU9J2xhYmVsJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlRleHRGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYmlsbGluZ2ludGVncmF0aW9uaGlzdG9yeXJlY29yZCcsCiAgICAgICAgICAgIG5hbWU9J2Rpc3BsYXlQcmlvcml0eScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5JbnRlZ2VyRmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J2JpbGxpbmdpbnRlZ3JhdGlvbmhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdsYWJlbCcsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5UZXh0RmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWx0ZXJGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYjJiYXBwcHJvbW9ibG9jaycsCiAgICAgICAgICAgIG5hbWU9J3RleHRWYXJpYW50JywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkNoYXJGaWVsZChjaG9pY2VzPVsoJ0JMQUNLJywgJ0JMQUNLJyksICgnQkxBQ0snLCAnQkxBQ0snKV0sIG1heF9sZW5ndGg9NTApLAogICAgICAgICksCiAgICBdCg==
```
Comment (sitozzz) [inline apps/condo/migrations/20221202114207-0195_acquiringintegration_displaypriority_and_more.js:2]: This is outdated migration name. You should recreate it after you open a final PR

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/miniapp/schema/AllMiniAppsService.js (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/schema/AllMiniAppsService.js:12]: Its common naming for operators and comparators, especially in backend-world :)
 `lhs` - left hand side, 
`rhs` - right hand side. 

So you can easily reinterpret args from `lhs < rhs`  to `operator<(lhs, rhs)` and so on

File: apps/condo/migrations/20221202114207-0195_acquiringintegration_displaypriority_and_more.js (hunk 2 above)
Comment (SavelevMatthew) [reply apps/condo/migrations/20221202114207-0195_acquiringintegration_displaypriority_and_more.js:2]: Yep, you're right...I'll have 2 or 3 total migrations....which I'll squash in final PR

**Pull request merged**